src/components/user/repo/pseudonym.repo.js

```diff
@@ -0,0 +1,22 @@
+const getService = (app) => {
+	return app.service('pseudonymModel');
+};
+
+const getPseudonyms = async (userId, app) => {
+	const pseudonyms = await getService(app).find({
+		query: {
+			userId,
+		},
+		paginate: false,
+	});
+	return pseudonyms._id ? Array.of(pseudonyms) : pseudonyms;
+};
+
+const deletePseudonyms = async (pseudonyms, app) => {
+	pseudonyms.forEach((pseudonym) => getService(app).remove(pseudonym._id));
```

Comment:
This will cause several synchronous calls, right?
Maybe we need to add `await` here.
```
const removePromises = pseudonyms.map((pseudonym) => getService(app).remove(pseudonym._id));
await Promise.all(removePromises);
```

Reply:
Done